app/assets/stylesheets/detailsPage.scss

```diff
@@ -80,6 +122,20 @@ h1 {
   text-align: center;
 }
 
+.tagHeader {
+  color: $gray-color;
+  font-weight: normal;
+  font-size: 12pt;
+  font-family: 'Arvo', sans-serif;
+  background-repeat:no-repeat;
```

Comment:
Should have a space after colon. There are several other places in the code that have this problem.

Reply:
Fixed everywhere